index.ts

```diff
@@ -114,6 +132,9 @@ export class RedisCache implements Cache {
     this.singleFlightGetCache = new Map();
     this.singleFlightApplyCache = new Map();
 
+    // Redis@3 & Redis@2 need to be promisified b/c they can't return promises (Redis@4 can)
+    // Ioredis already returns promises, so it returns the following warning when used
+    // (node:21449) [DEP0174] DeprecationWarning: Calling promisify on a function that returns a Promise is likely a mistake.
```

Comment:
Awesome commenting, love it

Reply:
Thanks. I got excited when I upgraded to node 18 and it went away but when I did it the right way the warnings stayed. Can address this another time (maybe when we upgrade redis version or remove it and only use ioredis)